addons/mail/static/src/models/activity/activity.js

```diff
@@ -283,6 +282,9 @@ function factory(dependencies) {
         creator: many2one('mail.user'),
         dateCreate: attr(),
         dateDeadline: attr(),
+        feedback: attr({
+            default: false,
+        }),
```

Comment:
Missing comment: field that contains backup feedback content of a mark-as-done in progress.

_(some other comments suggest something is wrong with being a backup in some specific flow, and suggestion is to have it being synced in real-time like composer text input)_